### ISSUE TITLE
New layout for improved scrolling

### ISSUE THREAD

**Issue by vilnius-leopold** [vilnius-leopold/big-band-dudel#9]
@Woruman I have changed the table layout to ease vertical and horizontal scrolling without loosing sight of the events header and the musicians sidebar. The new layout is currently online.

Right now this feature is on it's own feature branch. I would like to merge it into the master branch.

But before I do that, I would like to get your opinion on the new table layout.

**Comment by Woruman** [comment]
Am 22.06.2016 um 00:04 schrieb vilnius-leopold:

> @Woruman I have changed the table layout to ease vertical and horizontal scrolling without loosing sight of the events header and the musicians sidebar. The new layout is currently online.
> 
> Right now this feature is on it's own feature branch. I would like to merge it into the master branch.
> 
> But before I do that, I would like to get your opinion on the new table layout.

Am I right, that I can find it on the normal adress to test it?

Yeah, thank you for the feature. I really like to be able to see the
names all the time.

One thing, that would be cool, is, to be able to get a quick overview
without having to use a filter. (or the tilter has to be very easy to use)
Therefore:
- everything on top has to use as less space as possible (the dates). If
  possible, it has to move somewhere else (the Add-Buttons, the Entermode,
  the blue box)
- it would be cool, to be possble to scroll away the dates to get more
  space for the participants

if i use strg - to minimize the size of the site, I have only the
possiblities of shrinking it to much or not to be able to see every
participant.

I wonder if it would be usefull to rotate everything. so to have the
dates in the column and the names in a row. I think its much more
important to get a quick overview of the instrumentation as of all the
dates. So, if every internetbrowser allows more width than height I
think, this would be cool. perhabs with names rotated 90°.

btw: the blue box moves the button for the edit-mode

So in summery, I'd like to get a better overview, even if this is only
practicable for a band of our size. If there would be a band with much
more musicions, they would have to use a filter anyway.

**Comment by ETorfbinder** [comment]
> I wonder if it would be usefull to rotate everything.

Ich glaube das Tauschen von Reihe/Spalte verkompliziert die Sache dahingehend, dass man dann die Namen nicht mehr gut lesen kann - und um die geht es ja. Das Scollen find ich dagegen weniger störend, zumal ja nun die Namen und Events immer sichtbar sind. 

> - everything on top has to use as less space as possible (the dates). If possible, it has to move somewhere else (the Add-Buttons, the Entermode, the blue box)

...würde eher versuchen, da noch etwas mehr Platz zu gewinnen

> - it would be cool, to be possble to scroll away the dates to get more space for the participants

evtl. genügt schon eine gekürzte Anzeige (z.B. nur der Event-Name; Ausblenden von Mod, Datum, Ort)

Filter wären eine Möglichkeit, das ganze etwas übersichtlicher zu machen (Filtern bzw. sortieren nach: zuletzt gedudelt; Instrument; Name)

**Comment by vilnius-leopold** [comment]
@Woruman, @ETorfbinder ja, mehr platz zu gewinnen ist eine gute idee! Ist mir auch schon aufgefallen.

Ich denk mal darüber nach was sich gut umsetzen lässt

**Comment by Woruman** [comment]
Ok, das klingt logisch mit der schlechten Erkennbarkeit der
90°-gedrehten Namen. Schade, in die Breite wäre soviel Platz...

Scrollen ist halt von demher nervig, dass das Abstimmungsverhalten von
allen nicht mehr auf einen Blick erkennbar ist.

Das mit der Event-Ausblenden/gekürzt-Anzeigen-Funktion find ich gut!

Zum Filter:

Sinnvolle Funktionen wären:
- ordnen nach Instrumentengruppen
- ordnen nach Abstimmungsergebnis (erst alle jas, dann naguts, dann
  vlt.s dann neins, dann noch nicht abgestimmt...)
- Kombination von beidem möglich
- Ausblenden von Events/Auswählen von Events zur Anzeige (um mehr Platz
  zu gewinnen, bzw. nicht hin und her scrollen zu müssen)
- abstimmungszeitpunkt
- name
